### PR TITLE
feat(Fun Import): import remote service or function to local workspace

### PR DESCRIPTION
**Pull request by ChanDaoH**
re #10, #6

允许将云端资源导入到本地

image

image

image

image

image

### CODE REVIEW

File: src/commands/importFunction.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  vscode.commands.executeCommand(serverlessCommands.REFRESH_LOCAL_RESOURCE.id);
+}
+
+async function checkDir(serviceName: string, functionName: string): Promise<boolean> {
+  const rootPath = <string>cwd;
+  const serviceDirPath = path.join(rootPath, serviceName);
+  const funcDirPath = path.join(serviceDirPath, functionName);
+  if (isDirectory(serviceDirPath)
+    && isDirectory(funcDirPath)
+    && hasFileUnderDir(funcDirPath)) {
```
Comment (vangie) [inline src/commands/importFunction.ts]: hasFileUnderDir => isNotEmpty

File: src/commands/importService.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    if (choice === 'Cancel') {
+      return;
+    }
+  }
+  const { importService } = require('@alicloud/fun/lib/import/service.js');
+  await importService(serviceName, cwd).catch((ex: any) => { console.error(ex.message) });
+  vscode.commands.executeCommand(serverlessCommands.REFRESH_LOCAL_RESOURCE.id);
+}
+
+async function checkDir(serviceName: string): Promise<string[]> {
```
Comment (vangie) [inline src/commands/importService.ts]: checkDir => existLocalFunctionFiles

File: src/commands/importFunction.ts (hunk 1 above)
Comment (ChanDaoH) [reply src/commands/importFunction.ts]: 已更新

File: src/commands/importService.ts (hunk 2 above)
Comment (ChanDaoH) [reply src/commands/importService.ts]: 已更新

**Pull request merged**